src/notebook/indicators/reference_indicators.md

```diff
@@ -0,0 +1,957 @@
+    nb_pdc / (
+        SELECT 
+            sum(nb_pdc) 
+        FROM 
+            t1
+    ) * 100 AS pct_nb_pdc,
+    p_range
+FROM 
+    t1
+```
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+#### requête locale
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+Exemple requête sur la département (02) paris (75) 't2-02-75'
+
+```sql
+WITH 
+    t1 AS (
+    WITH 
+        puissance(p_range, p_cat) AS (
+            VALUES 
+                (numrange(0, 15.0), 1), 
+                (numrange(15.0, 26.0), 2), 
+                (numrange(26, 65.0), 3),
+                (numrange(65, 175.0), 4),
+                (numrange(175, 360.0), 5),
+                (numrange(360, NULL), 6)
+        )
+    SELECT
+        count(id_pdc_itinerance) AS nb_pdc,
+        p_range,
+        department.code
+    FROM
+        department,
+        pointdecharge 
+        LEFT JOIN station ON station.id = station_id 
+        LEFT JOIN localisation ON localisation_id = localisation.id
+        LEFT JOIN puissance ON puissance_nominale::numeric <@ p_range 
+    WHERE
+        department.code = '75'
+        AND ST_Within ("coordonneesXY", department.geometry)
+    GROUP BY
+        p_range,
+        department.code
+    ORDER BY
+        nb_pdc DESC
+        )
+SELECT
+    nb_pdc / (
+        SELECT 
+            sum(nb_pdc) 
+        FROM 
+            t1
+    ) * 100 AS pct_nb_pdc,
+    p_range,
+    code
+FROM 
+    t1
+```
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+### T3 : Nombre de stations par nombre de points de recharge
+
+'nb_stations' est le nombre de stations.
+
+'nb_pdc' est le nombre de pdc.
+
+ex. il y a 2790 stations (nb_stations) avec un seul pdc (nb_pdc).
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+#### requête globale
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+Exemple requête globale : 't3-00-00' ou 't3'
+
+```sql
+WITH 
+    stat_nb_pdc AS (
+    SELECT
+      count(station_id) AS nb_pdc,
+      localisation_id
+    FROM
+      pointdecharge
+      LEFT JOIN station ON station.id = station_id
+    GROUP BY
+      station_id,
+      localisation_id
+)
+SELECT
+    count(nb_pdc) AS nb_stations,
+    nb_pdc
+FROM
+    stat_nb_pdc
+    LEFT JOIN localisation ON localisation_id = localisation.id
+GROUP BY
+    nb_pdc
+ORDER BY
+    nb_stations DESC
+```
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+#### requête locale
+<!-- #endregion -->
+
+<!-- #region editable=true slideshow={"slide_type": ""} -->
+Exemple requête sur la commune (04) d'Annemasse (74012) 't3-04-74012'
+
+```sql
+WITH 
+    stat_nb_pdc AS (
+    SELECT
+      count(station_id) AS nb_pdc,
+      localisation_id
+    FROM
+      pointdecharge
+      LEFT JOIN station ON station.id = station_id
+    GROUP BY
+      station_id,
+      localisation_id
+)
```

Comment:
not required?

Reply:
Oui, c'est une solution provisoire en attendant d'avoir le champ nbre_pdc bien documenté.

Lorsqu'on compare les deux, l'écart est important.

```sql
WITH 
    stat_nb_pdc AS (
    SELECT
      count(station_id) AS nb_pdc,
      localisation_id,
      nbre_pdc
    FROM
      pointdecharge
      LEFT JOIN station ON station.id = station_id
    GROUP BY
      station_id,
      localisation_id,
      nbre_pdc
)
SELECT
    nb_pdc,
    nbre_pdc
FROM
    stat_nb_pdc
```